arch/arm/boot/dts/qcom/qcom-msm8974pro-oneplus-bacon.dts

```diff
@@ -180,6 +180,16 @@
 		interrupts-extended = <&tlmm 31 IRQ_TYPE_EDGE_FALLING>;
 		omit-battery-class;
 	};
+
+        nfc@28 {
+                compatible = "nxp,nxp-nci-i2c";
+                reg = <0x28>;
+
+                interrupts-extended = <&tlmm 59 IRQ_TYPE_EDGE_FALLING>;
```

Comment:
Other devices in mainline seem to use either `IRQ_TYPE_EDGE_RISING` or `IRQ_TYPE_LEVEL_HIGH`, so I'm inclined to think `IRQ_TYPE_EDGE_FALLING` is not very correct

Reply:
What should the touchscreen (rmi4) and charger (bq....) have to do with the nfc chip? I don't quite get why you look at those right now.